tests/unit/sagemaker/image_uris/regions.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import boto3
+
+
+def regions():
```

Comment:
i seem to recall seeing this method somewhere else

Reply:
in a recent PR of mine or somewhere else? I refactored the recently added one to this one so that it could be used across files. I did a grep for the botocore region-determination logic, and found only some code that generates EC2 regions in the Neo unit tests